dali/base/dadfs.cpp

```diff
@@ -941,14 +943,48 @@ class FileClusterInfoArray: public IArrayOf<IClusterInfo>
                     for (i=0;i<clustlist.ordinality();i++)
                         if (strcmp(clustlist.item(i),cname.str())==0)
                             break;
-                    if (i==clustlist.ordinality())
+                    if (i==clustlist.ordinality()) // JCS - this will always be true if made it this far??
                         clustlist.append(cname.str());
                 }
                 if (!*s)
                     break;
                 clusters = s+1;
             }
         }
+
+        unsigned done = 0;
+        StringBuffer clusterLabel;
```

Comment:
Minor: This code would be easier to understand if clustlist was named preferedclusterList or similar.

Reply:
agree, I'll change.